hermetic_build/DEVELOPMENT.md

```diff
@@ -78,7 +78,7 @@ as per [POSIX env var definition](https://pubs.opengroup.org/onlinepubs/96999197
 2. Move the jar into its well-known location.
 
    ```shell
-   mv /path/to/jar "${HOME}/.library_generation/gapic-generator-java.jar"
+   mv ~/.m2/repository/com/google/api/gapic-generator-java/{version}/gapic-generator-java-{version}.jar "${HOME}/.library_generation/gapic-generator-java.jar"
```

Comment:
Is this too much detail? What if a user change the default maven dir?

Reply:
I thought it may be more convenient for copying the whole line and limit to making a replacement in `{version}`. I think a non-default maven setup is not a common use case, but still checks out, so I reverted this line.